oap-server/server-configuration/configuration-zookeeper/pom.xml

```diff
@@ -0,0 +1,156 @@
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>server-configuration</artifactId>
+        <groupId>org.apache.skywalking</groupId>
+        <version>6.2.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>configuration-zookeeper</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.skywalking</groupId>
+            <artifactId>configuration-api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.skywalking</groupId>
+            <artifactId>library-client</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.curator</groupId>
+            <artifactId>curator-recipes</artifactId>
+        </dependency>
+    </dependencies>
+
+    <profiles>
+        <profile>
+            <id>CI-with-IT</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>io.fabric8</groupId>
+                        <artifactId>docker-maven-plugin</artifactId>
+                        <configuration>
+                            <sourceMode>all</sourceMode>
+                            <showLogs>true</showLogs>
+                            <logDate>default</logDate>
+                            <verbose>true</verbose>
+                            <imagePullPolicy>IfNotPresent</imagePullPolicy>
+                            <autoCreateCustomNetworks>true</autoCreateCustomNetworks>
+                            <images>
+                                <image>
+                                    <name>zookeeper:3.4.14</name>
+                                    <alias>zookeeper-dynamic-configuration-integration-test-zookeeper</alias>
+                                    <run>
+                                        <network>
+                                            <name>test-network</name>
+                                            <alias>zookeeper</alias>
+                                        </network>
+                                        <env>
+                                        </env>
+                                        <namingStrategy>none</namingStrategy>
+                                        <dependsOn>
+                                        </dependsOn>
+                                        <links>
+                                        </links>
+                                        <ports>
+                                            <port>zookeeper.port:2181</port>
+                                        </ports>
+                                        <wait>
+                                            <log>Zookeeper started successfully</log>
```

Comment:
The signal that zk starts successfully is not this line of log, it should be something like `binding to port /0.0.0.0:2181`